convert/src/test/kotlin/com/github/zeldigas/text2confl/convert/confluence/ReferenceProviderImplTest.kt

```diff
@@ -68,4 +69,49 @@ internal class ReferenceProviderImplTest {
 
         assertThat(result).isNotNull().isEqualTo(Xref("Sub Title One", "test"))
     }
+
+
+    @Test
+    internal fun `Http resolution`() {
```

Comment:
PR title says that there is some fix for windows. What actually is failing here on windows?